tests/tests_C2_economic_functions.py

```diff
@@ -0,0 +1,86 @@
+import src.C2_economic_functions as e_functions
+
+project_life = 20
+wacc = 0.1
+investment_t0 = 220000
+tax = 0.15
+lifetime = [20, 15]
+present_value = 295000
+fuel_keys = {
+    "fuel_price": 1.3,
+    "fuel_price_change_annual": 0,
+    "project_lifetime": project_life,
+    "wacc": wacc
+}
+
+
+def test_annuity_factor_calculation():
+    """
+
+    Tests whether the MVS is correctly calculating the annuity factor
+    """
+    AF = e_functions.annuity_factor(project_life, wacc)
+    assert AF == 1 / 0.1 - 1 / (0.1 * (1 + 0.1) ** 20)
+
+
+def test_crf():
+    """
+
+    Tests whether the MVS is correctly calculating the capital recovery factor
+    """
+    CRF = e_functions.crf(project_life, wacc)
+    assert CRF == (0.1 * (1 + 0.1) ** 20) / ((1 + 0.1) ** 20 - 1)
+
+
+def test_capex_from_investment(lifetime):
+    """
+
+    Tests whether the MVS is correctly calculating the capital expenditure of the project
+    """
+    for years in lifetime:
+        if years == project_life:
+            CAPEX = e_functions.capex_from_investment(investment_t0, years, project_life,wacc,tax)
+            assert CAPEX == 220000*(1+0.15)
+        else:
+            CAPEX = e_functions.capex_from_investment(investment_t0, years, project_life, wacc, tax)
+            first_investment = 220000*(1+0.15)
+            assert CAPEX == first_investment + first_investment / (1 + 0.1) ** (15 * 1) + first_investment \
+                   + first_investment / (1 + 0.1) ** (15 * 1) + first_investment / (1 + 0.1) ** (15 * 2) \
+                   - ((first_investment / ((1 + 0.1) ** 15)) / 15) * (2 * 15 - 20)
+
```

Comment:
@SabineHaas is this a good way to test that function, or can you think of any way to break the function down into smaller ones?

Reply:
I am not sure whether I understand what you mean. the function `annuity()` is already very short. 
Small note: could you make sure when referring to a piece of code that we can see the function name here? I had to search in the code for it..

Reply:
Ah okay - I even searched for the wrong piece of code. 
I've already suggested to make two (or now with your comment above three) tests out of this test function.